File: src/main/java/com/getlose/guess/Game.java
package com.getlose.guess;

import java.util.Scanner;

public class Game {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number =0;
        while(number != -1){
            System.out.print("Please enter a number:");
            number = scanner.nextInt();
            System.out.println(number);
        }

    }

}

File: src/main/kotlin/com/getlose/Game/Game.kt
package com.getlose.Game

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val secret = Random().nextInt(10)+1
    print("secret: "+secret)
    var number = 0

    while(number != secret){
        print("please enter a number:")
        number = readLine()!!.toInt()//readLine()?.toInt()//scanner.nextInt()
        if(secret>number){
            println("higher")
        }else if(secret < number){
            println("lower")
        }else{
            print("got it")
        }
    }


}